lookout/style/format/benchmarks/commit_changes.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+cd $1
+if [ -z $(git status --porcelain) ]]; then
+  echo "no changes"
+  exit 1
+fi
+git checkout -b $2 --quiet
+git commit -a -m $2 --quiet
+git rev-parse HEAD
+git revert HEAD --no-edit 1>&2
+git rev-parse HEAD
+git checkout master --quiet
+exit 0
```

Comment:
misses newline and it might be better to just remove `exit 0` altogether since it overrides the return value of something that should not fail.

Reply:
sorry, I do not get which return values it can override?

Reply:
in a script the last command's return value is returned (here the one of `git checkout master --quiet`). If this fails it's good info so just removing the exit is probably a net benefit.

Reply:
ok, got it